layers.go

```diff
@@ -1158,7 +1174,8 @@ func (s *store) newLayerStore(rundir, layerdir, imagedir string, driver drivers.
 			layersImageDir,
 			filepath.Join(volatileDir, "volatile-layers.json"),
 		},
-		layerdir: layerdir,
+		layerdir:        layerdir,
+		tempDirRootPath: s.graphRoot,
```

Comment:
Is this correct? Shouldn’t that be a dedicated directory?
Also, it needs to precisely match the directories of data which we are deleting (like `tspath`/`datadir`). `graphRoot` seems to amount to the correct filesystem now, with the current only caller, but tying it more precisely to the `layerStore` logic would be easier to maintain.